gradle/validation/validate-source-patterns.gradle

```diff
@@ -181,8 +252,13 @@ class ValidateSourcePatternsTask extends DefaultTask {
       }
     }
 
+    ProgressLogger progress = progressLoggerFactory.newOperation(this.class)
+    progress.start(this.name, this.name)
+
     sourceFiles.each{ f ->
-      logger.debug('Scanning source file: {}', f);
+      progress.progress("Scanning ${f.name}")
+      logger.info('Scanning source file: {}', f);
```

Comment:
I would keep this as debug or remove after adding the progress meter.
I often try to run gradlew with `--info`, but theis is way too much information.

Do you know how to silence the palantir version-lock checker? I hate that it prints so much information during configuration phase to info.

I'd like to enable `--info` on jenkins, but that's way to much.

Reply:
The --info switch is verbose. I don't think there's much you can do about it. I was thinking about it myself - perhaps we should devise our own internal logging verbosity level (and log to lifecycle when it's enabled).

Reply:
Info is like ant always was. Except the shitty palantir logging. Can we suppress this? I have all Gradle builds with info logging in jenkins, except Lucene.

Reply:
I don't know how to do that, Uwe.

Reply:
Nevertheless, I'd change ours back to debug (it's really only for debugging). If a violation happens, filename is printed anyways.

But we should change this now to not use relative paths anymore, because it's hard to figure out where file is located. I can take care of that.

Reply:
Hmm... What do you mean by "not use relative paths"?

Reply:
When it reports a violation it's relative to basedir of file collection. This was ok we with root level task, but should be changed now to just print full path.

Reply:
Doesn't the logger carry the information which project this is? This should be sufficient to locate the file in question? I don't mind changing it to an absolute path, but I think project + path should be enough to locate something?

Reply:
I just wanted to add this to this dicsussion: https://github.com/gradle/gradle/issues/1010
I am working on that!

(I also figured out that we miss our lockfactory integration test in `:lucene:core`, in ANT it's still there (many JVMs hammering a single directory with lockfactories). We need a task in gradle for this, too. I will open another issue)